td/tracking/forms.py

```diff
@@ -96,7 +96,8 @@ class Meta:
         widgets = {
             'created_by': forms.HiddenInput(),
             'start_date': SelectDateWidget(
-                attrs={'class': 'date-input'}
+                attrs={'class': 'date-input'},
+                years=range(datetime.datetime.now().year - 2, datetime.datetime.now().year + 11),
```

Comment:
This is going to run once when the module loads instead of when the form loads. Best to put logic like that in the `__init__` method